recipes/geomatics/LICENSE

```diff
@@ -0,0 +1,29 @@
+BSD 3-Clause License
```

Comment:
The name of this file should be added to `MANIFEST.in` so that the sdist on pypi compiles with:
>  Redistributions of source code must retain the above copyright notice, this
   list of conditions and the following disclaimer.